engines/scumm/detection.cpp

```diff
@@ -1329,6 +1331,21 @@ SaveStateDescriptor ScummMetaEngine::querySaveMetaInfos(const char *target, int
 	return desc;
 }
 
+static const ExtraGuiOption scummComiObjectLabelsOption = {
+	_s("Show Object Line"),
+	_s("Show the names of objects at the bottom of the screen"),
+	"object_labels",
+	true
+};
+
+const ExtraGuiOptions ScummMetaEngine::getExtraGuiOptions(const Common::String &target) const {
```

Comment:
You need to look at ``engines/metaengine.h``. ``engines/advancedDetector.h`` is not the generic interface (SCUMM doesn't even use the ``AdvancedDetector`` code).